src/style.css

```
@@ -0,0 +1,125 @@
+html {
+  font-family: "Roboto", Arial, sans-serif;
+  --elements-position: 16px;
+  --use-color: #00acdc;
+  --thin-color: #616070;
+}
+
+body {
+  margin: 0;
+  color: #060b35;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-self: center;
+  padding: 32px var(--elements-position) 0 var(--elements-position);
+  border: 1px solid #fff;
+}
+
+.card__foto {
+  display: flex;
+  margin-left: 3px;
+  width: 160px;
+  height: 134px;
+}
+
+.card .card__foto {
+  align-items: center;
+}
+
+.card__description-product {
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 40px 0 4px;
+}
+
+.card__description-code {
+  line-height: 14px;
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--thin-color);
+}
+
+.stars-viewed {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 16px 0 24px;
+}
+
+.viewed {
+  font-size: 10px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars:nth-child {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__price {
+  display: flex;
+  line-height: 18px;
+  text-align: center;
+  justify-content: space-between;
+}
+
+.card__price-price {
+  font-size: 12px;
+  color: var(--thin-color);
+}
+
+.card__price-count {
+  font-weight: 700;
+}
+
+.card__button {
+  justify-content: center;
+  text-align: center;
+  font-weight: 700;
+  font-size: 14px;
+  margin-top: var(--elements-position);
+  display: flex;
```

```suggestion
  margin-top: var(--elements-position);
  display: flex;
  justify-content: center;
  text-align: center;
  font-weight: 700;
  font-size: 14px;
```
remember about order, check in other places